Lib/tkinter/__init__.py

```diff
@@ -3103,16 +3109,14 @@ def insert(self, *args):
         self.tk.call((self._w, 'insert') + args)
 
     def itemcget(self, tagOrId, option):
-        """Return the resource value for an OPTION for item TAGORID."""
+        """Return the value of OPTION for item TAGORID."""
         return self.tk.call(
             (self._w, 'itemcget') + (tagOrId, '-'+option))
 
     def itemconfigure(self, tagOrId, cnf=None, **kw):
-        """Configure resources of an item TAGORID.
+        """Query or modify the configuration options of an item TAGORID.
```

Comment:
```suggestion
        """Query or modify the configuration options of item *tagOrID*.
```
Just remembered that

Reply:
https://peps.python.org/pep-0257/#multi-line-docstrings says "*Do not* use the Emacs convention of mentioning the arguments of functions or methods in upper case in running text.  Python is case sensitive and the argument names can be used for keyword arguments, so the docstring should document the correct argument names."  'tagOrID' is an example of why not.  It goes on to suggest " It is best to list each argument on a separate line. For example:..." but this tends to bloat the doc.  Don't add *s if you do not want, but please drop the all caps at least here.

Reply:
Using different convention for different methods will do more harm than good. This is a different issue, we will change the convention after fixing all docstrings.

Reply:
Yes, a separate issue/PR to change the entire file would be even better, and will gladly help.  I am resolving all the related suggestions and approving the PR as is.

Reply:
But it would be better first implement support of Markdown-like, reStructuredText-like, and TeX-like mark up in `pydoc` (and make it configurable per module).

Reply:
I fear this would overcomplicate pydoc. Perhaps we could implement a small subset, but I don't think the additional complexity of markup-in-docstrings is worth it?

Reply:
Yes, I meant a small subset.